src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,142 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private Object[] objects;
```

Comment:
use the parametrized type
```suggestion
    private T[] values;
```